### ISSUE TITLE
Introduction failure

### ISSUE THREAD

**Issue by GiganteX** [fluentassertions/fluentassertions#1544]
Tried the sample code from the Introduction at https://fluentassertions.com/introduction :+1: 

IEnumerable numbers = new[] { 1, 2, 3 };
numbers.Should().OnlyContain(n => n > 0);
numbers.Should().HaveCount(4, "because we thought we put four items in the collection");

... but got the error that NonGenericCollectionAssertions doesn not contain a definition for OnlyContain()...

Also it says the response should be "Expected numbers to contain 4 item(s) because we thought we put four items in the collection, but found 3."

I am assuming it's a mistake and that it's not smart enough to end the sentence like that...

**Comment by dennisdoomen** [comment]
You're right. `OnlyContain` only works on generic collections. In fact, in the upcoming 6.0 version we dropped support for non-generic collections. 

But the failure message is correct. That's a bit of text is that you get from the 2nd parameter of `HaveCount`

**Comment by dennisdoomen** [comment]
Should be fixed now after Github has rebuild the site.

**Comment by GiganteX** [comment]
Thanks!

"But the failure message is correct. "

Wow, that's neat!
